### PR TITLE
Stats - Add analytics tracking for Insights and Period Stats

### LINKED ISSUE #2917: Add analytics tracking for Stats Insights
Ref:  https://github.com/wordpress-mobile/WordPressCom-Analytics-iOS/pull/26

Events to add (in the iOS naming conventions):
```
WPAnalyticsStatStatsInsightsAccessed,
WPAnalyticsStatStatsPeriodDaysAccessed,
WPAnalyticsStatStatsPeriodWeeksAccessed,
WPAnalyticsStatStatsPeriodMonthsAccessed,
WPAnalyticsStatStatsPeriodYearsAccessed,
```

### PR DESCRIPTION
**Pull request by daniloercoli**
Fix #2917

/cc @astralbodies - used the event names we've agreed in chat.

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java [hunk 1]
```diff
@@ -218,6 +214,8 @@ public void run() {
                             }
                         }
                     }, StatsConstants.STATS_SCROLL_TO_DELAY);
+
+                    trackStatsAnalitycs();
```
Comment (astralbodies) [inline WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java]: Small spelling error.

File: libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerMixpanel.java [hunk 2]
```diff
@@ -564,6 +564,31 @@ private AnalyticsTrackerMixpanelInstructionsForStat instructionsForStat(
                 instructions.setSuperPropertyAndPeoplePropertyToIncrement("number_of_times_accessed_stats");
                 instructions.setCurrentDateForPeopleProperty("last_time_accessed_stats");
                 break;
+            case STATS_INSIGHTS_ACCESSED:
```
Comment (astralbodies) [inline libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerMixpanel.java]: I'll make sure to copy these Mixpanel values when I do a PR for the main WPiOS repo.

**Comment by astralbodies** [comment]
Looks good from my side :shipit:

File: libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerMixpanel.java [hunk 3]
```diff
@@ -564,6 +564,31 @@ private AnalyticsTrackerMixpanelInstructionsForStat instructionsForStat(
...
+                instructions = AnalyticsTrackerMixpanelInstructionsForStat.
+                        mixpanelInstructionsForEventName("Stats - Insights Accessed");
+                instructions.setSuperPropertyAndPeoplePropertyToIncrement("number_of_times_accessed_insights_screen_stats");
+                instructions.setCurrentDateForPeopleProperty("last_time_accessed_insights_screen_stats");
```
Comment (nbradbury) [inline libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerMixpanel.java]: Missing `break` statement here and for the next few cases.

**Comment by nbradbury** [comment]
This is good to merge once the missing `break` statements are added.

**Comment by nbradbury** [comment]
:shipit:

**Pull request merged**